playbooks/webapp/README.md

```diff
@@ -85,7 +85,7 @@ To delete the webapp:
 ### Common
 
 * **operation** (str): Operation for the webapp playbook to perform, either `create` or `delete`. Default: `create`
-* **resource_prefix** (str): (Required) A prefix to prepend to the name of all AWS resources created for the webapp
+* **resource_prefix** (str): A prefix to prepend to the name of all AWS resources created for the webapp. Default: `ansible-test`
```

Comment:
This modification should have no impact on current users of this playbook. I don't consider it a breaking change. I believed that setting a default value, rather than making it a required parameter, was a more practical approach.